### PR TITLE
CRM-21518: Added missing containers to template

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
Overview
----------------------------------------
This PR adds missing container classes in templates which are missing.

Screenshots
----------------------------------------
This PR does not modify anything of visual difference. The idea here is to allow any other theme (Shoreditch in this context or any other in future) to have unified UI/UX solution if they use such class.

Comments
----------------------------------------
This is tested with default styling of CiviCRM and custom theme Shoreditch

---

 * [CRM-21518: Add missing structure divs in templates](https://issues.civicrm.org/jira/browse/CRM-21518)

### CODE REVIEW

File: templates/CRM/Admin/Page/ContactType.tpl [hunk 1]
```diff
@@ -66,4 +68,5 @@
     {crmButton q="action=add&reset=1" icon="plus-circle"}{ts}Add Contact Type{/ts}{/crmButton}
     {crmButton p="civicrm/admin" q="reset=1" class="cancel" icon="times"}{ts}Done{/ts}{/crmButton}
   </div>
+</div>
 {/if}
```
Comment (colemanw) [inline templates/CRM/Admin/Page/ContactType.tpl]: It looks to me like the opening of the div tag is outside the {if} clause but the closing </div> tag is inside the {/if}

File: templates/CRM/Admin/Page/PdfFormats.tpl [hunk 2]
```diff
@@ -69,3 +70,4 @@
       {crmButton p="civicrm/admin" q="reset=1" class="cancel" icon="times"}{ts}Done{/ts}{/crmButton}
     </div>
 {/if}
+</div>
```
Comment (colemanw) [inline templates/CRM/Admin/Page/PdfFormats.tpl]: need a newline at the end of this file

**Review by colemanw — CHANGES_REQUESTED** [review]

File: templates/CRM/Admin/Page/PdfFormats.tpl (hunk 2 above)
Comment (mukeshcompucorp) [reply templates/CRM/Admin/Page/PdfFormats.tpl]: @colemanw added

File: templates/CRM/Admin/Page/ContactType.tpl (hunk 1 above)
Comment (mukeshcompucorp) [reply templates/CRM/Admin/Page/ContactType.tpl]: This was looking like it due to malformed spacing. I fixed it. Now it should look good.

**Review by colemanw — APPROVED** [review]

**Pull request merged**